bypros/proxy.go

```diff
@@ -208,6 +211,11 @@ func (s *Service) doCatchUP(outChan catchUpOut, stopChan chan bool, req *CatchUp
 
 	browseDone := make(chan struct{})
 
+	if req.UpdateEvery < 1 {
+		log.Warn("got invalid 'updateEvery' value, setting it to 1")
```

Comment:
Hmm - I would even return an error here. Except if you have some stable code out there that doesn't support this value yet. But if `updateEvery < 1` is invalid, that's an error...